code/game/objects/structures/plasticflaps.dm

```diff
@@ -7,6 +7,17 @@
 	anchored = 1
 	layer = 4
 
+/obj/structure/plasticflaps/CanAStarPass(ID, to_dir, caller)
+	. = 1 //diseases, stings, etc can pass
+
+	if(istype(caller, /mob/living))
+		if(istype(caller,/mob/living/simple_animal/bot/mulebot))
```

Comment:
> I don't believe return on it's own actually returns the contents of .

Take 10 seconds to test it, you'll believe it.

Anyway i'll change it to ```return 1``` as asked, no problem with that.